pkg/apis/eksctl.io/v1alpha5/iam.go

```diff
@@ -54,15 +70,23 @@ func (iamMeta *ClusterIAMMeta) AsObjectMeta() metav1.ObjectMeta {
 // ClusterIAMServiceAccount holds an IAM service account metadata and configuration
 type ClusterIAMServiceAccount struct {
 	ClusterIAMMeta `json:"metadata,omitempty"`
+
+	// list of ARNs of the IAM policies to attach
 	// +optional
 	AttachPolicyARNs []string `json:"attachPolicyARNs,omitempty"`
-	// AttachPolicy holds a policy document to attach to this service account
+
+	// attachPolicy holds a policy document to attach to this service account
```

Comment:
```suggestion
	// AttachPolicy holds a policy document to attach to this service account
```
Should correspond to the field name i think, otherwise it appears in the json schema description (or just remove it completely)